specutils/analysis/width.py

```diff
@@ -141,26 +213,51 @@ def _compute_gaussian_sigma_width(spectrum, regions=None):
 
 
 def _compute_single_fwhm(flux, spectral_axis):
+    """
+    This is a helper function for the above `fwhm()` method.
+    """
 
-    argmax = np.argmax(flux)
-    halfval = flux[argmax] / 2
+    # The .value attribute is used here as the following algorithm does not
+    # use any array operations and would otherwise introduce a relatively
```

Comment:
Do you really mean "any array operations" here, or rather "Quantity", because `flux.value` _is_ a `np.ndarray` and you are using indexing operations etc.?